<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SelectUserDialog</class>
 <widget class="QDialog" name="SelectUserDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>761</width>
    <height>354</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QTableWidget" name="UserTable">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>30</y>
     <width>331</width>
     <height>271</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="SendNoticeBtn">
   <property name="geometry">
    <rect>
     <x>590</x>
     <y>320</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Confirm</string>
   </property>
  </widget>
  <widget class="QPushButton" name="CancelBtn">
   <property name="geometry">
    <rect>
     <x>670</x>
     <y>320</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Cancel</string>
   </property>
  </widget>
  <widget class="QLabel" name="UserListLabel">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>331</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Users</string>
   </property>
  </widget>
  <widget class="QTableWidget" name="SelectedUserTable">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>30</y>
     <width>331</width>
     <height>271</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="SelectBtn">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>110</y>
     <width>41</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>&gt;</string>
   </property>
  </widget>
  <widget class="QPushButton" name="ReturnBtn">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>160</y>
     <width>41</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;</string>
   </property>
  </widget>
  <widget class="QLabel" name="SelectedUserListLabel">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>10</y>
     <width>331</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Selected users</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
